content/en-us/build/stateless/intro.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Build Stateless DApps"
+weight: 1001
+menu:
+  build:
+    parent: "phat-stateless"
+---
+
+## Why Stateless DApps
+
+Compared with traditional smart-contract-based DApps which store all their states on-chain and require transactions for interaction, the desired use cases of Phat Contract happen off-chain with no (or limited) data stored on-chain. For example, instead of implementing an [ERC-20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) token with Phat Contract (whose balance has to be stored on-chain), we recommend to deploy your ERC-20 contract on Ethereum, and use Phat Contract to operate it.
+
+Our insight here is that:
+
+> Only a very limited portion of your DApp data really needs to be stored on-chain.
+
+Since the on-chain storage can be expensive (from both monetary and performance perspectives), it's timeworthy to refactor your DApps and move the stateless parts to Phat Contract. The benefits include:
```

Comment:
"Refactor your DApps" will scare the developers. We want the developers to use Phat Contract to replace their centralized service, instead of replacing their existing dapps / contracts. I would promote the idea in this way:

- Phat Contract provides computation but no storage. It's easiest to build stateless program.
- But no worry, backend and especially serverless backend are usually stateless and they worked very well.
- As a complementary to smart contract, most of the use cases can play really well with stateless program
- If you have to use state, we have solutions

Reply:
Updated.